### ISSUE TITLE
The 'count' attribute on the cellXfs node is not set correctly in styles.xml

### ISSUE THREAD

**Issue by DM-Francis** [EPPlusSoftware/EPPlus#696]
The 'count' attribute on the `cellXfs` node does get set correctly in styles.xml when the ExcelPackage is saved.
See this code sample:

```csharp
using System.Drawing;
using OfficeOpenXml;
using OfficeOpenXml.Style;

ExcelPackage.LicenseContext = LicenseContext.Commercial;

var package = new ExcelPackage();
var worksheet = package.Workbook.Worksheets.Add("Sheet 1");

var range1 = worksheet.Cells["A1:C3"];
range1.Style.Fill.PatternType = ExcelFillStyle.Solid;
range1.Style.Fill.BackgroundColor.SetColor(Color.Yellow);

var range2 = worksheet.Cells["D1:F3"];
range2.Style.Fill.PatternType = ExcelFillStyle.Solid;
range2.Style.Fill.BackgroundColor.SetColor(Color.Blue);

var filePath = Path.Combine(Environment.CurrentDirectory, "EpplusStyleTest.xlsx");
package.SaveAs(filePath);
```
There are two styles that have been added, and this is the resulting `cellXfs` node in styles.xml:
```xml
    <cellXfs count="1">
        <xf numFmtId="0" fontId="0" xfId="0" applyProtection="1" />
        <xf numFmtId="0" fontId="0" fillId="2" applyFill="1" applyProtection="1" />
        <xf numFmtId="0" fontId="0" fillId="3" applyFill="1" applyProtection="1" />
    </cellXfs>
```
There are 3 child nodes, but the count is still 1.  The count should be showing 3 here.

I've had a look through the source code, and the issue looks to be in `ExcelStyles.UpdateNamedStylesAndXfs`, but I haven't yet figured out exactly what the fix should be.

The issue may also apply to the `cellStyleXfs` node.

Some additional information for context: I spotted this because we had a piece of code that was adding additional styles (by updating the xml directly) and it used the count attribute to figure out what the next Ids should be.  After upgrading from 4.5.3 to 6.0.6 the styles were broken, and I narrowed it down to this.   There is an easy workaround which is to count the child nodes rather than rely on the count attribute, but thought I'd raise this issue anyway!

**Comment by JanKallman** [comment]
I'll have a look at this and will add a fix shortly.

**Comment by JanKallman** [comment]
I have added a fix for this issue. You can try it using our develop build: https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-Develop-Nuget-Feed

**Comment by DM-Francis** [comment]
Thanks for looking at it so quickly!
I've tried running our code combined with the develop build and the `cellXfs` node all looks good now.  I think there may still be an issue with the count in the `cellStyleXfs` node, here's a sample from one of the excel sheets I've just generated:
```
<cellStyleXfs count="0">
    <xf numFmtId="0" fontId="0" fillId="0" borderId="0" applyProtection="1" />
</cellStyleXfs>
```
I don't have any sample code to re-create this one though, and its not impacting our codebase in anyway, but thought I'd flag.
